codec-memcache/src/main/java/io/netty/handler/codec/memcache/MemcacheMessage.java

```diff
@@ -21,22 +21,4 @@
  * Marker interface for both ascii and binary messages.
  */
 public interface MemcacheMessage extends MemcacheObject, ReferenceCounted {
-
-    /**
-     * Increases the reference count by {@code 1}.
-     */
-    @Override
-    MemcacheMessage retain();
-
-    /**
-     * Increases the reference count by the specified {@code increment}.
-     */
-    @Override
-    MemcacheMessage retain(int increment);
-
-    @Override
-    MemcacheMessage touch();
-
-    @Override
-    MemcacheMessage touch(Object hint);
```

Comment:
same comment as above. This is needed as otherwise it will not return the correct type and so not allow method chaining correctly.